academic_observatory_workflows/database/sql/create_ror.sql.jinja2

```diff
@@ -15,7 +15,7 @@
 # Author: Richard Hosking, James Diprose #}
 
 SELECT 
-  rors.*,
-  STRUCT(regions) as iso3166
+  rors.* EXCEPT (country),
+  country
 FROM `{{ project_id }}.{{ ror.dataset_id }}.ror{{ ror.release_date.strftime('%Y%m%d') }}` as rors
-LEFT JOIN `{{ project_id }}.{{ iso.dataset_id }}.iso3166_countries_and_regions` as regions ON (rors.country.country_code = regions.alpha2)
+LEFT JOIN `{{ project_id }}.{{ settings.dataset_id }}.country` as country ON (rors.country.country_code = country.alpha2)
```

Comment:
Apologies if you have tested this, but will it be able to still find the `rors.country.country_code` field? Because above you select `rors.* EXCEPT (country)`

Reply:
Yeah, it just means that the `rors.country` field is replaced entirely by the `country` row that it is joined with. It happens after the join.